src/app/invest/invest.component.ts

```diff
@@ -10,68 +9,55 @@ import { map, startWith, tap } from 'rxjs/operators'
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class InvestComponent {
+  investState = InvestState
 
-  project = new BehaviorSubject<InvestProjectModel>({
+  projectSub = new BehaviorSubject<InvestProjectModel>({
     title: "Supercool Invest-O-Pia",
-    walletBalance: 50000,
+    walletBalance: 500,
     roi: "8%",
     description: "This is a small change, but a big move for us. 140 was an arbitrary choice based on the 160 character SMS limit. Proud of how thoughtful the team has been in solving a real problem people have when trying to tweet. And at the same time maintaining our brevity, speed, and essence!",
-    minInvestment: 10000,
-    maxInvestment: 100000,
-    imgSrc: "https://images.pexels.com/photos/2850347/pexels-photo-2850347.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260"
+    minInvestment: 100,
+    maxInvestment: 1000,
+    imgSrc: "https://images.pexels.com/photos/2850347/pexels-photo-2850347.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
```

Comment:
As you see, I've changed every amount to normal scale (1 == $1, not 100 == $1 anymore). This is because we are now dealing with the real scale values on all places (with one exception on assets, but we'll handle it later). This way, we'll me less prone to bugs related to multiplying and dividing with 100 :)